src/classes/CW_BriefApprovalProcessTest.cls

```diff
@@ -0,0 +1,297 @@
+/**
+* File Name   :    CW_BriefApprovalProcessTest.cls
+* Description :    B-106804 As a Marketing Ops I want the ability to submit a MarCom Brief
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 24/04/2014 Cristina P.	Intial
+*
+*/
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.